src/js/view/controls/components/chapters.mixin.js

```diff
@@ -2,11 +2,14 @@ import { ajax } from 'utils/ajax';
 import srt from 'parsers/captions/srt';
 
 class Cue {
-    constructor (time, text) {
+    constructor (time, text, cueType) {
         this.time = time;
         this.text = text;
         this.el = document.createElement('div');
         this.el.className = 'jw-cue jw-reset';
+        if (cueType && typeof cueType === 'string') {
+            this.el.className += ` jw-cue-type-${cueType}`;
+        }
```

Comment:
Can you do this to a string variable before setting `className`? Setting `className` more than once could impact performance by triggering repaints or reflow. Ex:
```suggestion
        let cssClasses = 'jw-cue jw-reset';
        if (cueType && typeof cueType === 'string') {
            cssClasses += ` jw-cue-type-${cueType}`;
        }
        this.el.className = cssClasses;
```

Reply:
awesome idea